### ISSUE TITLE
PVWatts snow model DC power loss due to snow

### ISSUE THREAD

**Issue by cpaulgilman** [NREL/SAM#1137]
### Describe the bug

When snow model is enabled in pvwattsv8, DC snow loss = 0 for time steps with snow loss:

![image](https://user-images.githubusercontent.com/27711933/186465697-d8f7adaf-100c-4a4f-b3b1-7c67ee137c6b.png)

And, DC snow loss = 1 for time steps with gen > 0, regardless of whether the snow model is enabled:

![image](https://user-images.githubusercontent.com/27711933/186465938-ddd3346b-17b8-49a3-997a-635b847d7e16.png)

### To Reproduce

1. Create a PVWatts / No Financial case.
2. Change the weather file to Fargo, ND (legacy TMY2 file with snow data).
3. Click **Simulate**.
4. Note DC power loss due to snow (%) value of 1 for time steps with System power generated (kW) > 0 (and some sunup or sundown time steps).
5. On System Design page, expand Advanced Inputs panel and check **Estimate snow losses**.
6. Click **Simulate**.
7. Note gen = 0 for first few days, but DC snow loss = 0, also true for Feb 1 when next snow event occurs.

### Expected behavior

Expect DC power loss due to snow to be a percentage of the array DC output, such as 100% when snow completely covers the module at the beginning of a snow event.

### Operating System and Version

 - OS: Windows 10
 - Version: SAM 2021.12.02 r2

**Comment by sjanzou** [comment]
fixed in ssc pull request https://github.com/NREL/ssc/pull/874
